fetools-app/src/pages/UnitConverter.jsx

```diff
@@ -37,6 +48,7 @@ function UnitConverter() {
     const newPixels = newEm * basePixelSize;
     setPixels(newPixels);
     setTailwindSize(newEm * 4);
```

Comment:
We need to either display nothing or display the em size in brackets here when no matching tailwind size exists. I know you are on top of this, just calling it out here to make sure we keep it on the radar.